source/MaterialXTest/MaterialXRender/RenderUtil.cpp

```diff
@@ -363,13 +363,18 @@ bool ShaderRenderTester::validate(const mx::FilePathVec& testRootPaths, const mx
                                 usedImpls.insert(nodeGraphImpl ? nodeGraphImpl->getName() : impl->getName());
                             }
 
-                            const mx::StringSet& wedgeFiles = options.wedgeFiles;
                             const mx::StringVec& wedgeParameters = options.wedgeParameters;
                             const mx::FloatVec& wedgeRangeMin = options.wedgeRangeMin;
                             const mx::FloatVec& wedgeRangeMax = options.wedgeRangeMax;
                             const mx::IntVec& wedgeSteps = options.wedgeSteps;
+                            const mx::StringVec& wedgeFiles = options.wedgeFiles;
+                            mx::StringSet wedgeFileSet;
+                            if (!wedgeFiles.empty())
+                            {
+                                wedgeFileSet.insert(wedgeFiles.begin(), wedgeFiles.end());
+                            }
```

Comment:
Nitpick: I think you could do the set construction in one line: `mx::StringSet wedgeFileSet(wedgeFiles.begin(), wedgeFiles.end());`